tests/publiccloud/ssh_interactive_end.pm

```diff
@@ -16,7 +16,7 @@ use utils;
 sub run {
     my ($self, $args) = @_;
     select_host_console(force => 1);
-    $args->{my_provider}->cleanup($args);
+    $args->{my_provider}->cleanup($args) unless check_var('PUBLIC_CLOUD_SLES4SAP', 1);
```

Comment:
but this way you would also skip `terraform destroy` call :thinking:

Reply:
yes, sap test are handling it with a dedicated test module named `tests/sles4sap/publiccloud/qesap_cleanup.pm`

Reply:
ok